src/calibrate-multi-hop/controllers/calibrate.py

```diff
@@ -0,0 +1,55 @@
+import app
+from routes import api
+from flask import Blueprint, request, jsonify
+import logging
+import app
+import json
+import os
+from pymongo import MongoClient
+from models import calibrate as cb
+
+
+calibrate_bp = Blueprint('calibrate_bp', __name__)
+
+
+MONGO_URI = os.getenv('MONGO_URI')
+client = MongoClient(MONGO_URI)
+db = client['airqo_netmanager_staging_airqo']
+col = db['calibration_ratios']
+
+@calibrate_bp.route(api.route['ratios'], methods=['POST', 'GET'])
```

Comment:
I am one of those who is cautious about the usage of these HTTP verbs, could there be any reason why we allow both POST and GET on the same endpoint? Just need some enlightenment....could it be that we want to trigger 2 different actions based on HTTP verbs used in the request? Or do we just want to use both POST and GET on the same implementation? Just curious....

Reply:
What could be the drawback of using/allowing one? That is, either POST only or GET only?

Reply:
Not sure - @Priscilla-AL?